packages/react-nexus/src/useNexus.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import invariant from 'ts-invariant';
+import { NexusClient } from '@bbp/nexus-sdk';
+import nexusContext from './nexusContext';
+
+const warningMessage =
+  'No Nexus client found. ' +
+  'To use nexus-react components, make sure you wrap your React app with the NexusProvider component' +
```

Comment:
"nexus-react"? Isn't it called "react-nexus"?

Reply:
:+1: